ui/app/src/components/EditSiteDialog/EditSiteDialogContainer.tsx

```diff
@@ -22,19 +22,27 @@ import { isBlank } from '../../utils/string';
 import { update } from '../../services/sites';
 import { fetchSites } from '../../state/reducers/sites';
 import { EditSiteDialogContainerProps } from './utils';
-import { updateEditSiteDialog } from '../../state/actions/dialogs';
-import { batchActions } from '../../state/actions/misc';
+import {
+  closeSingleFileUploadDialog,
+  showSingleFileUploadDialog,
+  updateEditSiteDialog
+} from '../../state/actions/dialogs';
+import { batchActions, dispatchDOMEvent } from '../../state/actions/misc';
 import { showErrorDialog } from '../../state/reducers/dialogs/error';
 import { ConditionalLoadingState } from '../SystemStatus/LoadingState';
 import { EditSiteDialogUI } from './EditSiteDialogUI';
 
+const siteImagePath = `/.crafter/screenshots/default.png?crafterSite=`;
 export function EditSiteDialogContainer(props: EditSiteDialogContainerProps) {
   const { site, onClose, onSaveSuccess, isSubmitting } = props;
   const [submitDisabled, setSubmitDisabled] = useState(false);
   const sites = useSelector<GlobalState, LookupTable>((state) => state.sites.byId);
   const dispatch = useDispatch();
   const [name, setName] = useState(site.name);
   const [description, setDescription] = useState(site.description);
+  const [siteImageCounter, setSiteImageCounter] = useState(0);
+  const [siteImage, setSiteImage] = useState(`${siteImagePath}${site.id}`);
```

Comment:
make sure the fallback is in place. just like the sites grid, when there's no pic, it should show the fallback.